docs/developer/discounts/manual-discounts.mdx

```diff
@@ -0,0 +1,346 @@
+                },
+                "lines": [
+                    {
+                        "id": "T3JkZXJMaW5lOmFlYTBmNjMzLWU0MTItNDkwZS1hODhhLThlMTg3YzQ3N2ZkNQ==",
+                        "quantity": 2,
+                        "undiscountedTotalPrice": {
+                            "net": {
+                                "amount": 100.0
+                            }
+                        },
+                        "totalPrice": {
+                            "net": {
+                                "amount": 80.0
+                            }
+                        },
+                        "undiscountedUnitPrice": {
+                            "net": {
+                                "amount": 50.0
+                            }
+                        },
+                        "unitPrice": {
+                            "net": {
+                                "amount": 40.0
+                            }
+                        },
+                        "unitDiscount": {
+                            "amount": 10.0
+                        }
+                    },
+                    {
+                        "id": "T3JkZXJMaW5lOmU3ZWY1YjY4LTUzYmMtNGIyOC1iMmU2LWI0YjI3OWE0NTg2OA==",
+                        "quantity": 1,
+                        "undiscountedTotalPrice": {
+                            "net": {
+                                "amount": 30.0
+                            }
+                        },
+                        "totalPrice": {
+                            "net": {
+                                "amount": 30.0
+                            }
+                        },
+                        "undiscountedUnitPrice": {
+                            "net": {
+                                "amount": 30.0
+                            }
+                        },
+                        "unitPrice": {
+                            "net": {
+                                "amount": 30.0
+                            }
+                        },
+                        "unitDiscount": {
+                            "amount": 0.0
+                        }
+                    }
+                ]
+            }
+        }
+    }
+}
+```
+As a result, the discount reduced line 1 unit price from 50$ to 40$. 
+
+### Manual order discount
+The manual order discount can be applied with [`orderDiscountAdd`](api-reference/orders/mutations/order-discount-add.mdx) mutation.
+It impacts both the subtotal and shipping price. When the order qualifies for an order promotion or has an entire order voucher applied, 
+the manual order discount takes precedence and overrides them. It's important to note that the discount is calculated based on [base subtotal](developer/price-calculation.mdx#discount-application) 
+and [base shipping](developer/price-calculation.mdx#discount-application) prices, meaning [line-level](developer/discounts/overview.mdx#level) discounts are not overriden. 
```

Comment:
so does it mean they are accumulated? Maybe we should write a paragraph how it works when both line and order discounts are applied